content/en/docs/_index.md

```diff
@@ -29,5 +35,13 @@ Operators must claim ownership of their APIs. OLM will prevent conflicting Opera
 ### Declarative UI controls
 Operators can behave like managed service providers. Their user interface on the command line are APIs. For graphical consoles OLM annotates those APIs with descriptors that drive the creation of rich interfaces and forms for users to interact with the Operator in a natural, cloud-like way. 
 
+
+## Understanding sample OLM use-cases to get started.
+
+Operator Lifecycle Manager (OLM) helps users install, update, and manage the lifecycle of all Operators and their associated services running across their clusters.
+Operator solves this problem and basically replaces this "human" operator with a "software" operator. At its core it has the same control loop mechanism that Kubernetes has, that watches for changes in the application state.
+
+- For developers, a self-service experience allows provisioning and configuring instances of databases, monitoring without having to be subject matter experts, because the Operator has that knowledge baked into it.
```

Comment:
This doesn't really give me a sample use case for using OLM.

Reply:
Thank you for the comment. I am working on improving the article.